database/factories/ModelFactory.php

```diff
@@ -48,55 +61,67 @@
             ])->id;
         },
         'northstar_id' => $this->faker->northstar_id,
-        'url' => $url,
```

Comment:
So a plain `post` will default to a `text` post since we don't include a `url`?

Reply:
Oh wait I see now that this is like a base `post` and won't be used on its own will it?

Reply:
That's the idea -- this came from a chat @DFurnes and I were having over in https://github.com/DoSomething/rogue/pull/836#discussion_r365403495.